eq-author/src/App/page/Preview/CalculatedSummaryPreview.test.js

```diff
@@ -2,6 +2,7 @@ import React from "react";
 import { shallow } from "enzyme";
 import { render, flushPromises } from "tests/utils/rtl";
 import actSilenceWarning from "tests/utils/actSilenceWarning";
+import commentsSubscription from "../../../components/CommentsPanel/commentSubscription.graphql";
```

Comment:
```suggestion
import commentsSubscription from "components/CommentsPanel/commentSubscription.graphql";
```

All folders in `eq-author/src/` can be imported like that. Not the same for `eq-author-api` and `eq-publisher` though.